### DISCUSSION TITLE
How to copy contents of left window to the right window

### DISCUSSION THREAD

**Discussion by ildar-shaymukhametov** [neovim/neovim#27178] [Q&A]
![image](https://github.com/neovim/neovim/assets/74532770/521e47a9-8919-4e1a-a32e-cec10ee6f576)
I have a split window, I want to replace the left window with the contents of the right window, so that both windows have same file opened?
How can this be done? Is there a convenient shortcut like Ctrl-W Ctrl-Something?

**Comment by Sirivasv** [comment]
Thinking of duplicating the window by opening a new window with the same file with `Ctrl-W Ctrl-S` and closing the previous one
some other available commands can be found here https://neovim.io/doc/user/windows.html#opening-window

**Comment by ildar-shaymukhametov** [comment]
Thanks!
